### PR TITLE
RD2: Send Campaign Update to Elevate RD

### PR DESCRIPTION
**Pull request by andrewyu-salesforce**
- Allow Campaign update for Elevate Recurring Donation and reflect the change to Elevate
# Critical Changes

# Changes

 - When you change the Campaign on an Elevate-connected Recurring Donation, it's reflected in Elevate.

# Issues Closed

# Community Ideas Delivered

# Features Intended for Future Release

# Features for Elevate Customers

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: src/classes/PS_Request.cls [hunk 1]
```diff
@@ -77,6 +77,7 @@ public inherited sharing class PS_Request {
     public static String ENDPOINT_ADD_TO_CAPTURE_GROUP = '/v1/payments/verified/batch/{0}/add';
     public static String ENDPOINT_CHARGE_CAPTURE_GROUP = '/v1/payments/verified/batch/{0}/capture';
 
+    public static String PRODUCT_METADATA_SCHEMA_URI = 'https://payments-js.elevate.salesforce.org/schema/productMetadata/donation-v1.1.0';
```
Comment (andrewyu-salesforce) [inline src/classes/PS_Request.cls:80]: This is a hardcoded URL according to Payment Team. The URL will work and only needs to be updated when we need the new productMetadata feature.

File: src/labels/CustomLabels.labels [hunk 2]
```diff
@@ -1721,6 +1721,14 @@
...
         <value>Please enter a new Effective Date. The Effective Date for an Elevate Recurring Donations cannot be in the future.</value>
     </labels>
+    <labels>
+        <fullName>RD2_ElevateRDCampaignCannotUpdateToNull</fullName>
+        <categories>Recurring-Donations</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Checks if Elevate Recurring Donation&apos;s Campaign update is valid or not</shortDescription>
+        <value>Please enter a Campaign. The Campagin for an Elevate Recurring Donations cannot update to null.</value>
+    </labels>
```
Comment (andrewyu-salesforce) [inline src/labels/CustomLabels.labels:1731]: @mattwashington @sam-knox not sure who to tag, but can you review the new label.

**Comment by andrewyu-salesforce** [comment]
Work Item: https://gus.lightning.force.com/lightning/r/ADM_Sprint__c/a0lB00000027RpCIAU/view

File: src/labels/CustomLabels.labels (hunk 2 above)
Comment (lparrott) [reply src/labels/CustomLabels.labels:1731]: My suggestion: "Please enter a Campaign. Once entered, the Campaign for an Elevate Recurring Donation can be changed, but not removed."

**Review by lparrott — APPROVED** [review]
Code looks good and the new feature works great! Just waiting on the error text, but that shouldn't hold up testing.

File: src/labels/CustomLabels.labels (hunk 2 above)
Comment (sam-knox) [reply src/labels/CustomLabels.labels:1731]: Here's what I'd suggest:

"You can't remove the Campaign on an active Elevate-connected Recurring Donation. To remove the Campaign, close the Recurring Donation and create a new one."

**Pull request merged**